tee-worker/scripts/litentry/release/release.sh

```diff
@@ -0,0 +1,336 @@
+ StandardOutput=file:${WORKER_DIR}/${LOG}
+ StandardError=inherit
+
+ [Install]
+ WantedBy=default.target
+ "
+   local service_filename="${service_name}.service"
+   echo "$service_template" > "$service_filename"
+   echo "Service file \"${service_filename}\" generated successfully."
+}
+
+generate_worker_service_file() {
+  local command="/bin/bash -c  'cd ${WORKER_DIR}/tmp/w0 && ./integritee-service --clean-reset -P 2000 -w 2001 -r 3443 -h 4545 --running-mode mock --enable-mock-server --parentchain-start-block 0 run --skip-ra --dev'"
+  local service_name='worker'
+  local description='Worker Service for Litentry Side chain'
+  local working_directory='/usr/local/bin'
+
+  generate_service_file_for_worker "${service_name}" "${description}" "${command}" "${working_directory}"
+}
+
+generate_upgrade_worker_service_file() {
+  echo "Latest Finalized block: ${LATEST_FINALIZED_BLOCK}"
+  local command="/bin/bash -c 'cd ${WORKER_DIR}/tmp/w0 && ./integritee-service -P 2000 -w 2001 -r 3443 -h 4545 --running-mode mock --enable-mock-server --parentchain-start-block ${LATEST_FINALIZED_BLOCK} run --skip-ra --dev'"
+  local service_name='worker'
+  local description='Worker Service for Litentry Side chain'
+  local working_directory='/opt/worker/'
+
+  generate_service_file_for_worker "${service_name}" "${description}" "${command}" "${working_directory}"
+}
+
+generate_parachain_service_file() {
+  local command="/bin/bash -c 'source /home/faisal/.nvm/nvm.sh && cd ${PARACHAIN_SOURCE} && scripts/launch-local-binary.sh rococo && sleep infinity'"
+  local service_name='litentry-parachain'
+  local description='Parachain Setup for Litentry'
+  local working_directory=$PARACHAIN_SOURCE
+
+  generate_service_file "${service_name}" "${description}" "${command}" "${working_directory}"
+
+}
+
+current_mrenclave(){
+  # TODO: Correct Working Directory
+  output=$(make mrenclave 2>&1)
+  if [[ $? -eq 0 ]]; then
+      mrenclave_value=$(echo "$output" | awk '{print $2}')
+      echo "MRENCLAVE value: $mrenclave_value"
+      export OLD_MRENCLAVE="$mrenclave_value"
+  else
+      echo "Failed to extract MRENCLAVE value."
+  fi
+}
+
+enclave_account(){
+  log=$(cd bin && ./integritee-service signing-key 2>&1)
+  enclave_account=$(echo "$log" | awk '/Enclave account:/{print $NF}')
+  if [[ -n $enclave_account ]]; then
+      echo "Enclave account value: $enclave_account"
+      export ENCLAVE_ACCOUNT="$enclave_account"
+      echo "ENCLAVE_ACCOUNT exported successfully."
+  else
+      echo "Failed to extract Enclave account value."
+  fi
+}
+
+new_mrenclave(){
+  output=$(make mrenclave 2>&1)
+  if [[ $? -eq 0 ]]; then
+      mrenclave_value=$(echo "$output" | awk '{print $2}')
+      echo "MRENCLAVE value: $mrenclave_value"
+      export NEW_MRENCLAVE="$mrenclave_value"
+  else
+      echo "Failed to extract MRENCLAVE value."
+  fi
+}
+
+latest_sync_block(){
```

Comment:
Is it sidechain block? If so it's better to explicitly say it in the function name -  and this function seems to do more than getting the sidechain block. More importantly it sets the scheduled update block I believe

Reply:
Updated in new script `deploy.sh`